tools/ci/src/commands/compile_check_no_std.rs

```diff
@@ -70,6 +70,14 @@ impl Prepare for CompileCheckNoStdCommand {
             "Please fix compiler errors in output above for bevy_math no_std compatibility.",
         ));
 
+        commands.push(PreparedCommand::new::<Self>(
+            cmd!(
+                sh,
+                "cargo check -p bevy_tasks --no-default-features --features edge_executor --features critical-section --target {target}"
```

Comment:
```suggestion
                "cargo check -p bevy_tasks --no-default-features --features edge_executor,critical-section --target {target}"
```

Reply:
Ahh ok, I had the separate flags before because I was including a space, thanks for the tip here!